config/setupTests.ts

```diff
@@ -0,0 +1,12 @@
+import '@testing-library/jest-dom';
+
+jest.mock('react', () => {
+  const originalModule = jest.requireActual('react');
+
+  return {
+    __esModule: true,
+    ...originalModule,
+    useLayoutEffect: jest.requireActual('react').useEffect,
+  };
+});
+Element.prototype.scrollTo = () => {};
```

Comment:
I feel like this kind of code should be part of the React testing library itself, but if we needed, let's keep it.